app/views/tenants/projects/index.html.erb

```diff
@@ -12,6 +12,11 @@
         <% project.categories.each do |category| %>
           <div class="project-category"><%= category.name %></div>
         <% end %>
+        <%= form_for :pending_loan, url: pending_loans_path do |f| %>
+          <%= f.hidden_field :project_id, value: project.id %>
+          <%= f.hidden_field :loan_amount, value: project.price %>
```

Comment:
The user doesn't need to see it, but it should be in the params.  Jeff and I looked over this and we both still agree that the field should be hidden.